src/Integration.Vsix/CFamily/PreCompiledHeaders/PreCompiledHeadersEventListener.cs

```diff
@@ -105,6 +104,16 @@ public void Dispose()
             {
                 activeDocumentTracker.OnDocumentFocused -= OnActiveDocumentFocused;
                 activeDocumentTracker?.Dispose();
+
+                try
+                {
+                    pchFilesDeleter.DeletePchFiles();
+                }
+                catch
```

Comment:
It's a good practice to follow the convention of only catching non-critical exceptions.